graders/r/r_autograder/pltest.R

```diff
@@ -54,7 +56,8 @@ result <- tryCatch({
         cat("[pltest] done question_details\n\n")
     }
 
-    print(list.files("bin/"))
+    print("test")
+    print(list.files("../student/"))
```

Comment:
Maybe protect behind the `debug` flag?

Reply:
Yes in favour of removing.  

We do turn debug on every now and then and don't want to flood the output too much :)